src/Views/EthernetPage.vala

```diff
@@ -30,14 +30,20 @@ namespace Network.Widgets {
         }
 
         construct {
+            widgets_stack = new Gtk.Stack ();
+            widgets_stack.visible = true;
+
+            var no_cable = new Granite.Widgets.AlertView (_("Cable unplugged"), _("No cable connected"), "");
             info_box.halign = Gtk.Align.CENTER;
 
-            top_revealer = new Gtk.Revealer ();
+            var top_revealer = new Gtk.Revealer ();
             top_revealer.valign = Gtk.Align.START;
             top_revealer.transition_type = Gtk.RevealerTransitionType.SLIDE_DOWN;
             top_revealer.add (info_box);
 
-            content_area.add (top_revealer);
+            widgets_stack.add_named (no_cable, "unplugged");
+            widgets_stack.add_named (top_revealer, "plugged");
```

Comment:
It's actually faster to use refs directly than to look up strings, so it would probably be better to scope `no_cable` and `top_revealer` to the class as private variables instead of using `add_named` and comparing names